app/src/main/java/io/github/droidkaigi/confsched/MainApplication.java

```diff
@@ -52,7 +52,9 @@ public void onCreate() {
                 .appModule(new AppModule(this))
                 .build();
 
-        Fabric.with(this, new Crashlytics());
+        if (!BuildConfig.DEBUG) {
```

Comment:
Oh, current Deploygate apk is debug build version.
So now this should be removed.
I will do after merge.